spoc/models.py

```diff
@@ -19,10 +20,10 @@
         "align_base_qscore": pa.Column(int),
         "pass_filter": pa.Column(bool),
     },
-    coerce=True
+    coerce=True,
 )
 
-annotated_fragment_schema = pa.DataFrameSchema(
+AnnotatedFragmentSchema = pa.DataFrameSchema(
```

Comment:
Maybe replace `"sister_identity"` with `"label"`? For the sake of generalization. Then we can make a subclass for SisterPoreC that checks valid values of `"label"`.

Reply:
That is a good thought! But I think there are already some files that depend on this, so I would postpone this until we generalize in all places and bump versions to systematically rerun analyses.